topics/go/algorithms/strings/reverse/reverse.go

```diff
@@ -0,0 +1,30 @@
+package strings
+
+func ReverseString(str string) string {
+
+	// Here we create a slice of runes.
```

Comment:
Don't use we, just state what you are doing.

// Creating a slice of runes from the string that was provided. UTF-8 is a character set based on runes (or code points).
// One to many runes make up the characters in the string. This code will assume one rune per character.